cw_bitcoin/lib/electrum_wallet.dart

```diff
@@ -2054,6 +2077,21 @@ abstract class ElectrumWalletBase
         final address = addressFromOutputScript(out.scriptPubKey, network);
 
         if (address.isNotEmpty) outputAddresses.add(address);
+
+        // Check if the script contains OP_RETURN
+        final script = out.scriptPubKey.script;
+        if (script.contains('OP_RETURN')) {
+          final index = script.indexOf('OP_RETURN');
+          if (index + 1 <= script.length) {
+            try {
+              final opReturnData = script[index + 1].toString();
+              final decodedString = utf8.decode(HEX.decode(opReturnData));
+              outputAddresses.add('OP_RETURN:$decodedString');
+            } catch (_) {
+              outputAddresses.add('OP_RETURN:');
+            }
+          }
+        }
```

Comment:
we can use `ElectrumTransactionInfo.fromBundle` since we are updating it with the same way